softwerkskammer/lib/mailsender/views/mailtemplate.pug

```diff
@@ -45,10 +45,13 @@ html
           text-decoration: none;
           vertical-align: middle;
           white-space: nowrap;
-        }  
+        }
   body
     | !{content}
-    
+
     each button in buttons
       a.btn(href=button.url) #{button.text}
-    
```

Comment:
Wie oben (plaintext). Mehr Leerzeilen, kein manueller Umbruch.`<hr>` statt `--`

```
    br
    br
    br
    hr
    p.
     Versendet über die E-Mail-Infratruktur des Hosters #[a(href='https://deltatree.de/') DeltaTree]

Reply:
> Wie oben (plaintext). Mehr Leerzeilen, kein manueller Umbruch.`<hr>` statt `--`
> 
> ```
>     br
>     br
>     br
>     hr
>     p.
>      Versendet über die E-Mail-Infratruktur des Hosters #[a(href='https://deltatree.de/') DeltaTree]
> ```

Ok, finde in der HTML-Variante `<hr>` auch besser, passe ich an.